app/models/edition/related_policies.rb

```diff
@@ -1,6 +1,16 @@
 module Edition::RelatedPolicies
   extend ActiveSupport::Concern
 
+  def delete_policy(content_id)
+    self.edition_policies.delete(EditionPolicy.where(policy_content_id: content_id))
```

Comment:
Could this be simplified to just `EditionPolicy.where(policy_content_id: content_id).delete_all`? I'm not sure this needs to be done via the `edition_policies` association.